freqtrade/misc.py

```diff
@@ -114,3 +115,23 @@ def deep_merge_dicts(source, destination):
             destination[key] = value
 
     return destination
+
+
+def find_project_root(path=None):
```

Comment:
see notes in #2221 about this function ...

Reply:
removed